ui/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "blackduck-alert",
-    "version": "6.11.0-SIGQA1",
```

Comment:
Shouldn't this have been updated by the 'Next Snapshot' stage? Is this a bug in the build library?

Reply:
The gradle task to build the UI updates the version to the gradle version which is updated with the Next Snapshot stage in the build.